M2/Macaulay2/packages/RationalMaps.m2

```diff
@@ -49,7 +49,7 @@ export{
 
 needsPackage("FastLinAlg");
```

Comment:
This belongs in the `PackageImports` option to `newPackage`.

Reply:
Done.